docs/TESTS.md

```diff
@@ -0,0 +1,38 @@
+# Tests
+If you have downloaded an exercise using the exercism CLI, you will have a folder containing the solution function template (which you then edit to create your solution), a test function and some metadata.
+
+## Running Tests
+Create a link in the active workspace to the exercise folder:
+
+```
+      ]LINK.Create # /path/to/exercise
```

Comment:
@rikedyp if users are working on more than one exercise within the session, do we need to either direct them to use a non-default workspace/namespace or show them how to `]Link.Break`? Trying to re-link `#` to a different folder errors with `Cannot link a non-empty namespace to a non-empty directory`.